docs/source/certs_management.md

```diff
@@ -0,0 +1,381 @@
+
+**Client:** X.509 certificates that allow signed requests to be passed from clients to Fabric nodes. These requests require a signature for admin level certificates.
+
+
+#### TLS Certificates
+
+TLS certificates allow Fabric nodes and clients to sign and encrypt communications. A valid TLS Certificate is required for any channel communication.
+
+
+### Enrollment Certificate Expiration
+
+All Enrollment certificates are assigned an expiration date by the issuing Certificate Authority (CA).  Expiration dates must be monitored, and certificates must be re-enrolled before expiration. The most important Enrollment Certificate parameter is the **Not After** element, which indicates its expiration date.
+
+## Certificates and Locations
+
+Organization CAs supply X.509 Enrollment Certificates for identities and the TLS CAs supply TLS Certificates for securing node and client communications.
+
+
+### Organization CA Certificates
+
+Organization CA Root certificates and Organization CA Admin certificates are issued for each organization.
+
+#### Organization CA Root Certificate
+
+**Description**: Public certificate that permits verification of all certificates issued by the Organization CA.
+
+**Location**: Stored on disk in the Organization CA directory, or in the channel configuration (ca-cert.pem).
+
+**Impact if expired**: A new Organization CA Root certificate must be issued. Organization CA Root certificates are valid for 15 years.
+
+
+#### Organization CA Admin Certificate
+
+**Description**: Certificate used when making admin requests to the Organization CA.
+
+**Location**: Dependent on implementation:
+
+<pre>
+msp
+ ├── IssuerPublicKey
+ ├── IssuerRevocationPublicKey
+ ├── cacerts
+ │   └── localhost-7053.pem
+ ├── keystore
+ │   └── key.pem
+ ├── signcerts
+ │   └── cert.pem
+ └── user
+</pre>
+
+**Impact if expired**: Cannot register or enroll new identities, but transaction traffic does not stop.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-orderer-org-s-ca-admin)
+
+
+### TLS CA Certificates
+
+TLS CA Root certificates and TLS CA Admin Certificates are issued for each organization.
+
+
+#### TLS CA Root Certificate
+
+**Description**: Certificate used when making admin requests to the TLS CA.
+
+**Location**: Stored on disk in the TLS CA directory or in the channel configuration (ca-cert.pem).
+
+**Impact if expired**: A new TLS CA Root certificate must be issued. TLS CA Root certificates are valid for 15 years.
+
+
+#### TLS CA Admin Certificate
+
+**Description**: Certificate used for admin requests to the TLS CA.
+
+**Location**: Dependent on implementation:
+
+<pre>
+msp
+ ├── IssuerPublicKey
+ ├── IssuerRevocationPublicKey
+ ├── cacerts
+ │   └── localhost-7053.pem
+ ├── keystore
+ │   └── key.pem
+ ├── signcerts
+ │   └── cert.pem
+ └── user
+</pre>
+
+**Impact if expired**: Cannot register or enroll new identities, but transaction traffic does not stop.
```

Comment:
'identities' in this context is confusing as people generally think of identities as coming from the Organization CA.
We need to explain that TLS certificates are registered and enrolled in the same way as Organization identities. That is, the result of a TLS enrollment is a TLS certificate.

Reply:
Sure, that doesn't really describe the impact though. Such as "All communication between the node and its channels stops." @jkneubuh ? 

I think this one was just missed by the review, unintended.

Reply:
Added to Key Concepts, an entry for TLS cert - "**TLS** - A public Transport Layer Security (TLS) certificate that authorizes client and node communications. On Fabric, registration and enrollment are the same for identity certificates and TLS certificates.

Reply:
@jkneubuh both seem good thanks.